website/docusaurus.config.js

```diff
@@ -539,6 +539,10 @@ module.exports = {
                         from: '/topics/feature-flags/tutorials/nextjs/implementing-feature-flags',
                         to: '/feature-flag-tutorials/nextjs/implementing-feature-flags',
                     },
+                    {
+                        from: '/topics/feature-flags/tutorials/astro/implementing-feature-flags',
+                        to: '/feature-flag-tutorials/astro/implementing-feature-flags',
```

Comment:
This redirect is not needed. We only add redirect when we move an exiting page that might have links from external domains we do not control.